Integrations/AutofocusV2/AutofocusV2.yml

```diff
@@ -15,6 +15,10 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Additional malicious verdicts
```

Comment:
@kirbles19 need to rename the display name of the additional AF categories to mark as malicious.

current mapping:

{
    'benign': 1,
    'malware': 3,
    'grayware': 2,
    'phishing': 3,
    'c2': 3
}

for now, only grayware (or benign but why :) ) can be marked also as malicious (similar to the threshold in other DBot cases) but theorertically it can be a list of categories